force-app/main/default/classes/DMLOperations.cls

```diff
@@ -145,9 +140,33 @@ public with sharing class DMLOperations {
      * @param oppNames The list of Opportunity names.
      */
     public static void upsertOpportunities(String accountName, List<String> oppNames) {
-        // Loop through each Opportunity name
-            // Create an Opportunity with the given name
-            // Add the Opportunity to the list
+        List<Opportunity> opportunityList = [SELECT Id,Name FROM Opportunity WHERE Name In :oppNames]; // Create a list to hold the existing Opportunities    
+        List<Opportunity> oppsToProcess = new List<Opportunity> (); // Create a list for the Opps to upsert
+        List<Account> accountsByName = [SELECT Id FROM Account WHERE Name = :accountName]; // Query to see if the Account exists
+        Account oppAccount = new Account (); // Create the Account variable
+
+        if (accountsByName.isEmpty() == false) {
```

Comment:
If you are checking a boolean return method, you can put `!` in front of it to check for false. `!accountsByName.isEmpty()`